### DISCUSSION TITLE
Question about Pseudo-bulk Replicates part in full manual

### DISCUSSION THREAD

**Discussion by Hachi6004** [GreenleafLab/ArchR#1329] [Questions / Documentation] [answered]
Hi,
Thanks for providing such helpful manual!

I have a question about the example in illustrating the process of pseudo-bulk replicate generation.
https://www.archrproject.com/bookdown/how-does-archr-make-pseudo-bulk-replicates.html

For 9.1.3 Cluster3, Rep3=250 cells, which is less than minCells(300), but the manual says "we can make one additional replicate with more than minCells". I am confused, because Rep3 is the "one additional replicate" right? But Rep3 only has 250 cells, less than 300, how can we say it is more than minCells (300)?

**Comment by rcorces** [comment, marked as answer]
This is a typo. Thanks for pointing it out.

```
Rep1 = 900 cells from SampleA
Rep2 = 400 cells from SampleB
Rep3 = 250 cells [100 cells from SampleC + 50 from SampleD + 50 from SampleE + 50 from SampleF]
```

should be ("100 from SampleF")

```
Rep1 = 900 cells from SampleA
Rep2 = 400 cells from SampleB
Rep3 = 300 cells [100 cells from SampleC + 50 from SampleD + 50 from SampleE + 100 from SampleF]
```

Given the table of cluster residence:
```
Sample  Cluster1  Cluster2  Cluster3  Cluster4  Cluster5
A       800       600       900       100       75
B       1000      50        400       150       25
C       600       900       100       200       50
D       1200      500       50        50        25
E       900       100       50        150       50
F       700       200       100       100       25
```

@paupaiz - can you make this change in the dev version of the documentation you are working on?

**Reply by paupaiz** [reply]
done!

**Reply by Hachi6004** [reply]
Thank you!!
